docs/building.adoc

```diff
@@ -1,34 +1,35 @@
 = Building the ADB Vagrant Box
-:toc:
 
-The Vagrant box is based on CentOS7. It is built using the latest
+The Vagrant box is based on CentOS 7. It is built using the latest
 packages from CentOS core. Some components of the ADB will be delivered
 in containers and will use code drawn from upstream projects to add
-newer features. Today no containers are including a fully built box. The
+newer features. Today no containers include a fully built box. The
```

Comment:
I feel "Today no containers include a fully built box" is grammatically incorrect . Should it be "Today there are no docker images included in the fully built box"